frontend/views/containers/Welcome.vue

```diff
@@ -1,33 +1,26 @@
 <template lang="pug">
-callout-card(
-  :isCard='true'
-  :svg='SvgHello'
-)
-  i18n.is-title-3.c-title(
-    tag='h3'
-    data-test='headerNeed'
-    :args='{ userName: userDisplayName ? userDisplayName : ourUsername }'
-  ) Welcome {userName}
-
-  div
+  callout-card(
+    :isCard='true'
+    :title='L("Welcome, {username}!", { username: ourUserDisplayName })'
+    :svg='SvgHello'
+  )
     i18n(tag='p') Add your income details to start receiving or giving mincome.
-
     i18n(
       tag='button'
-      data-test='openIncomeDetailModal'
+      data-test='openIncomeDetailsModal'
       @click='openModal("IncomeDetails")'
     ) Add income details
 </template>
 
 <script>
 import sbp from '~/shared/sbp.js'
-import { mapGetters } from 'vuex'
 import { OPEN_MODAL } from '@utils/events.js'
+import { mapGetters } from 'vuex'
 import CalloutCard from '@components/CalloutCard.vue'
 import SvgHello from '@svgs/hello.svg'
 
 export default {
-  name: 'StartInviting',
+  name: 'AddIncomeDetailsWidget',
```

Comment:
This should be the same as the filename. So either this needs to be `Welcome` or the file should be renamed

Reply:
File renamed to `AddIncomeDetailsWidget`.